telemetry/qml/TelemetryPermissionDialog.qml

```diff
@@ -168,7 +168,7 @@ Rectangle {
                         horizontalCenter: parent.horizontalCenter
                     }
 
-                    text: qsTr("For more information, please take a look at our <a href=\"https://musescore.com/legal/privacy\">Privacy Policy</a>")
+                    text: qsTr("For more information, please take a look at our %1Privacy Policy%2").arg("<a href=\"https://musescore.com/legal/privacy\">").arg("</a>")
```

Comment:
Why is this? Those "%1" "%2" can lead to confusion on transifex.

Reply:
It is the same we do elsewhere when URLs are involved

Reply:
And vital for security reasons, to prevent malicious translations using a different UIRL

Reply:
See e.g. https://github.com/musescore/MuseScore/blob/master/mscore/file.cpp#L176-L188

Reply:
Oh yes, I didn't think of that.